scripts/azure-pipelines/windows/run-tests.yml

```
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: MIT
+#
+# Builds and runs vcpkg-test.exe for PRs and CIs.
+
+jobs:
+- job: windows_tests
+  pool:
+    name: ${{ parameters.poolName }}
+  steps:
+  - task: CmdLine@2
+    displayName: "Build and Test vcpkg"
+    inputs:
+      script: |
+        :: TRANSITION, get these tools on the VMs next time we roll them
+        call .\bootstrap-vcpkg.bat
+        .\vcpkg.exe fetch cmake
+        .\vcpkg.exe fetch ninja
+        set PATH=downloads\tools\cmake-3.14.0-windows\cmake-3.14.0-win32-x86\bin;downloads\tools\ninja-1.10.0-windows;%PATH%
+        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat" -arch=x86 -host_arch=x86
+        rmdir /s /q build.x86.debug > nul 2> nul
+        cmake.exe -G Ninja -DCMAKE_BUILD_TYPE=Debug -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=ON -DVCPKG_DEVELOPMENT_WARNINGS=ON -DVCPKG_WARNINGS_AS_ERRORS=ON -DVCPKG_BUILD_FUZZING=ON -B build.x86.debug -S toolsrc

```

```suggestion
        cmake.exe -G Ninja -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=ON -DVCPKG_DEVELOPMENT_WARNINGS=ON -DVCPKG_WARNINGS_AS_ERRORS=ON -DVCPKG_BUILD_FUZZING=ON -B build.x86.debug -S toolsrc
```

Also, why are you building fuzzing? just to make sure it builds?

---

Yes.